### DISCUSSION TITLE
Bloated package with unused libraries

### DISCUSSION THREAD

**Discussion by wafendy** [serverless/serverless#10027] [General]
Hi,

Is there a way to reduce the package size by excluding the libraries that we do not used. For example, we are working on AWS platform, can we exclude all other libraries not related to AWS?

**Comment by pgrzesik** [comment]
Hello @wafendy - could you be a little bit more specific here? What libraries do you have in mind? Why would they be included if you're not using them in any way? Are they added to your `package.json` for example?
